### PR TITLE
Update message-rate-limits.md

### PR DESCRIPTION
**Pull request by Achuthan-Muguntharajan**
Have changes the line for message rate source  from :
*Message rate source**: This indicates how the message submission rate is calculated. Valid values are: <br/> `User`: The rate is calculated for sending users (specified with the **MAIL FROM** SMTP command)  to 
`User`: The rate is calculated for sending user (Based on how user authenticates in the SMTP session)

The reason being as follows . I work with Premier customer and this came up as one of the request and it was not evaluating based on mail from 
Test results so far :
=============
Connector setting :
Message Ratelimit :1
messageratesource:user

Scenario 1:
==========
Submitting emails anonymously  with different from addresses we were able to send only 1 message 
ex
 
Send-MailMessage -From t1@edgelab.com -To u16@edgelab.com -Subject 'Sending test' -Body "Hi and bye" -SmtpServer 'mbx.edgelab.com'
Send-MailMessage -From t2@edgelab.com -To u16@edgelab.com -Subject 'Sending test' -Body "Hi and bye" -SmtpServer 'mbx.edgelab.com'

Exchange 2010/2013/2016 behaviors:

This first one went through and the second one did not 
Second message failed with error message : Send-MailMessage : Service not available, closing transmission channel. The server response was: 4.4.2 Messagesubmission rate for this client has exceeded the configured limit

I also reviewed a detailed extrace:

==
SmtpReceive | SmtpInSession(id=11) cmd: MAIL FROM:<t1@edgelab-com>
-- | --
SmtpReceive | The   connection for user [Anonymous] with IP address [192.168.2.100] is being   dropped. The rate of message submissions from this connection has   exceeded the throttling policy of 1 per minute.
==
The point where the request failed it evaluated the user as anonymous  and blocked the email  although the mail from was different .  So this was first instance where we can observer it i snot evaluated based on mail from 

To further tested and  performed below scenario 

Using an account and giving send as permission on 2 mailbox and sending the message in the following way 
Send-MailMessage -From t1@edgelab.com -To u16@edgelab.com -Subject 'Sending test' -Body "Hi and bye" -SmtpServer 'mbx.edgelab.com' -credential $t3
Send-MailMessage -From t2@edgelab.com -To u16@edgelab.com -Subject 'Sending test' -Body "Hi and bye" -SmtpServer 'mbx.edgelab.com' -credential $t3

If it is evaluated based on mail from the second message must have gone through  but it did not proving that it is evaluated against how the user is authenticated in session

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeServer/mail-flow/message-rate-limits.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ The following table shows the message throttling options that are available on R
...
+|**Message rate source**: This indicates how the message submission rate is calculated. Valid values are: <br/> `User`: The rate is calculated for sending user (Based on how user authenticates in the SMTP session)<br/>• `IPAddress`: The rate is calculated for sending hosts. <br/>• `All`: The rate is calculated for both sending users and sending hosts.| `IPAddress` on the following default Receive connectors: <br/>• Default _\<ServerName\>_ in the Transport service on Mailbox servers. <br/>• Default Frontend _\<ServerName\>_ in the Front End Transport service on Mailbox servers. <br/>• Outbound Proxy Frontend _\<ServerName\>_ in the Front End Transport service on Mailbox servers. <br/>• Default internal Receive connector _\<ServerName\>_ on Edge Transport servers. <br/> `User` on the following default Receive connectors: <br/>• Client Proxy _\<ServerName\>_ in the Transport service on Mailbox servers. <br/>• Client Frontend _\<ServerName\>_ in the Front End Transport service on Mailbox servers.|Cmdlet: **New-ReceiveConnector** and **Set-ReceiveConnector** <br/> Parameter: _MessageRateSource_|Not available|
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/mail-flow/message-rate-limits.md:87]:
```suggestion
|**Message rate source**: This indicates how the message submission rate is calculated. Valid values are: <br/> `User`: The rate is calculated for sending user (based on how user authenticates in the SMTP session).<br/>• `IPAddress`: The rate is calculated for sending hosts. <br/>• `All`: The rate is calculated for both sending users and sending hosts.| `IPAddress` on the following default Receive connectors: <br/>• Default _\<ServerName\>_ in the Transport service on Mailbox servers. <br/>• Default Frontend _\<ServerName\>_ in the Front End Transport service on Mailbox servers. <br/>• Outbound Proxy Frontend _\<ServerName\>_ in the Front End Transport service on Mailbox servers. <br/>• Default internal Receive connector _\<ServerName\>_ on Edge Transport servers. <br/> `User` on the following default Receive connectors: <br/>• Client Proxy _\<ServerName\>_ in the Transport service on Mailbox servers. <br/>• Client Frontend _\<ServerName\>_ in the Front End Transport service on Mailbox servers.|Cmdlet: **New-ReceiveConnector** and **Set-ReceiveConnector** <br/> Parameter: _MessageRateSource_|Not available|
```

Comment (JohanFreelancer9) [reply Exchange/ExchangeServer/mail-flow/message-rate-limits.md:87]: Please lower case the "B" in (Based on how user authenticates in the SMTP session) and insert a period after the closing bracket.

**Comment by Achuthan-Muguntharajan** [comment]
> Copy review - minor edits.

Thanks for the suggestion . committed the suggested edits

**Comment by scanum** [comment]
@msdmaguire PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**